src/any_llm/types/completion.py

```
@@ -144,7 +144,79 @@ class ChatCompletionChunk(OpenAIChatCompletionChunk):
+            and key.startswith(("prompt_cache_", "cache_"))
+            and isinstance(meter, int)
+        }
+        if read is None and creation is None and not ttl and not meters:
+            return value
+        value = dict(value)
+        value["cache_usage"] = {
+            "read_input_tokens": read,
+            "creation_input_tokens": creation,
+            "creation_5m_input_tokens": ttl.get("ephemeral_5m_input_tokens"),
+            "creation_1h_input_tokens": ttl.get("ephemeral_1h_input_tokens"),
+            "included_in_prompt_tokens": True if cached is not None else False,
```

Ruff (0.16.1)

[warning] Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)